release.yml

```diff
@@ -113,5 +113,4 @@ components:
 
 cves:
```

Comment:
This needs to be removed. The `cves` section doesn't exist anymore, it's retrieved from JIRA based on the `version` field value.